Projects/AppCore/DesignSystem/Sources/Foundation/Fonts/Font.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  Font.swift
+//  AppCoreKit
+//
+//  Created by 마경미 on 10.06.24.
+//  Copyright © 2024 tellingUs. All rights reserved.
+//
+
+import UIKit
+
+public enum Font {
+    public enum Style {
+        case display_01_B
+        case display_01_R
+        case display_02_B
+        case display_02_R
+        case head_01_B
+        case head_01_R
+        case head_02_B
+        case head_02_R
+        case head_03_B
+        case head_03_R
+        case body_01_B
+        case body_01_R
+        case body_02_B
+        case body_02_R
+        case caption_01_B
+        case caption_01_R
+        case caption_02_B
+        case caption_02_R
```

Comment:
이거 폰트 이름에 번호가 달려있네용 !? 으음 🤔 뭔가 이상한데 디자이너분께 말씀드려야할 것 같은데.. 굳이인가 싶기도 하고... 

그런데 _ 뒤에 B R 이것은 weight 구분인 것 같은데 이것은 따로 enum타입으로 선언해주면 나중에 조립해서 사용하기에 더 편할 것 같은 생각이 들어요 ㅎㅎ body / display / head / caption 또한 4가지 종류 밖에 없는데 중복되는 코드가 많이 사용되어서 사용되는 style종류가 뭔지 찾기 어려운 감이 없지 않아 있기도 하고용 ㅎㅎ 

style / weight 를 가지고 있는 Font가 struct타입이고 폰트이름에 숫자가 아니라 크기라고 하면 아래처럼 깔끔하게 정의할 수도 있을 것 같아요 ~ 

```suggestion
    public enum Style: String {
        case display
        case head
        case body
        case caption
    }
    public enum Weight: String {
        case bold = "B"
        case regular = "R"
    }
    
    public enum Size: CGFloat {
        case 32
        case 28
        case 24
        case 20
        case 18
        case 16
        case 14
        case 12
        case 10
    }

    private var fontName: String { 
        return "\(style.rawValue)_"\(size.rawValue)"_"(weight.rawValue)"
    }
```

Reply:
Font구조체로 하게되더라도 다시 case를 저만큼 작성해줘야하지 않나용?
setText시에 한번에 .body_01_B 이런식으로 하려면요? setText에 body_01이랑, bold 이렇게 따로 받고싶으신건가용?

Reply:
아아 네~ weight랑 style은 다른개념이구 또 사이즈도 다른 개념이니 세 개를 다르게 선언하고 
폰트이름들은 bodyLarge bodyMedium bodySmall 등으로 통일하면 어떨까했어요~ ㅎㅎ 

그런데 이렇게 제안하면 디자이너분들도 수정하기 귀찮으니까 그대로 가요~